azurerm/config.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/arm/containerservice"
```

Comment:
Same on portal, when something goes wrong with autoscale settings, it just says "bad request". :(